func_sig_registry/utils/abi.py

```diff
@@ -23,6 +23,9 @@ def make_4byte_signature(text_signature):
     from .encoding import force_bytes
     return keccak_256(force_bytes(text_signature)).digest()[:4]
 
+def make_32byte_signature(text_signature):
+    from .encoding import force_bytes
+    return keccak_256(force_bytes(text_signature)).digest()
```

Comment:
I'd suggest removing this utility function and using [`eth_utils.event_signature_to_log_topic`](https://eth-utils.readthedocs.io/en/latest/utilities.html#event-signature-to-log-topic-event-signature-bytes)